Support.md

```diff
@@ -0,0 +1,25 @@
+# Support
+
+## Xamarin Supported Open Source Components
+
+Xamarin Supported Open Source components are a collection of Xamarin built bindings and libraries.
+
+| Name                                  | Description                                                                      | Source                                                           |
+|---------------------------------------|----------------------------------------------------------------------------------|------------------------------------------------------------------|
+| Android Support Libraries             | Bindings for Google's Android Support Libraries                                  | [GitHub](https://github.com/xamarin/AndroidSupportComponents)    |
+| Google Play Services Client Libraries | Bindings for Google's Play Services Client Libraries                             | [GitHub](https://github.com/xamarin/GooglePlayServicesComponents)|
+| Google API's for iOS                  | Bindings for Google's API's for iOS Libraries                                    | [GitHub](https://github.com/xamarin/GoogleAPIsForiOSComponents)  |
+| Facebook SDK's                        | Bindings for Facebook's iOS & Android SDK's                                      | [GitHub](https://github.com/xamarin/FacebookComponents)          |
+| Xamarin.Auth                          | Cross-platform API for authenticating users and storing their accounts.          | [GitHub](https://github.com/xamarin/Xamarin.Auth)                |
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
+feature request as a new Issue.
+
+For help and questions about using these libraries, please ask questions on [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/dotnet).
+
+## Microsoft Support Policy  
+
+Support for the libraries hosted in this repository are limited to the resources listed above.
```

Comment:
The resources listed above are NOT hosted in this repository !

Reply:
:)

Reply:
Let me update.